src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -33,21 +33,14 @@ public LineResponse saveLine(LineRequest request) {
         Station upStation = stationService.findById(request.getUpStationId());
         Station downStation = stationService.findById(request.getDownStationId());
         Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
-        return LineResponse.of(persistLine, stations);
+
+        return LineResponse.of(persistLine);
     }
 
     public List<LineResponse> findLines() {
```

Comment:
```suggestion
    @Transactional(readOnly = true)
    public List<LineResponse> findLines() {
```

조회만 있는 경우  readOnly 옵션을 추가해보면 어떨까요? 😄

Reply:
엇 `readOnly`를 놓쳤네요! 추가하겠습니다~~